controllers/auth.js

```diff
@@ -12,7 +12,11 @@ const authService = require("../services/authService");
 const githubAuth = (req, res, next) => {
   let userData;
   const rdsUiUrl = new URL(config.get("services.rdsUi.baseUrl"));
-  const authRedirectionUrl = req.query.state ?? rdsUiUrl;
+  let authRedirectionUrl = req.query.state ?? rdsUiUrl;
+
+  if (process.env.NODE_ENV === "development" && !req.query.state) {
+    authRedirectionUrl = "/healthcheck";
+  }
```

Comment:
❓Why are we doing this? Is this absolutely necessary?

Reply:
So when someone sets up their backend locally and tries to sign in with github into their backend, as there is no query state it redirects the user to the root path which gives a 404 and that has confused a lot of people in thinking that they have done something wrong. 
This will fix that.